generator/sbpg/targets/resources/sbp_messages_mod.rs

```diff
@@ -22,19 +22,19 @@ use self::(((p.identifier|mod_name)))::(((m.identifier|camel_case)));
 ((*- endfor *))
 use self::unknown::Unknown;
 
-#[cfg(feature = "sbp_serde")]
-use serde::{Serialize, Deserialize};
```

Comment:
Note: need to import this is removed because we're referring to the full path instead